app/forms/exceptions.py

```diff
@@ -0,0 +1,29 @@
+from rest_framework import status
+from rest_framework.exceptions import APIException
+
+
+class APIDuplicateSubmissionException(APIException):
+    status_code = status.HTTP_409_CONFLICT
+    default_detail = "Spørreskjemaet tillater kun én innsending"
+
+
+class DuplicateSubmission(ValueError):
+    pass
```

Comment:
Hva er formålet med å ha en klasse som inheriter, men ikke legger til noe mer? Har sett det flere ganger, men jeg er usikker på hensikten.

Reply:
Er for å mappe til riktig status-kode i tillegg, right @Eirsteir ?

Reply:
Ok skjønner. Lettere å se problemet med gode forklaringer og klassenavn